--- jemalloc-5.2.1/jemalloc.pc.in
+++ jemalloc-5.2.1-new/jemalloc.pc.in
@@ -7,6 +7,6 @@
 Name: jemalloc
 Description: A general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support.
 URL: http://jemalloc.net/
-Version: @jemalloc_version_major@.@jemalloc_version_minor@.@jemalloc_version_bugfix@_@jemalloc_version_nrev@
+Version: @jemalloc_version_major@.@jemalloc_version_minor@.@jemalloc_version_bugfix@
 Cflags: -I${includedir}
 Libs: -L${libdir} -ljemalloc${install_suffix}

