Author: Victor Roemer
Date: February 23, 2023

RTLD_DEEPBIND is exclusive to Glibc
diff --git a/src/vlib/unix/plugin.c b/src/vlib/unix/plugin.c
index 1260089f4..99f509004 100644
--- a/src/vlib/unix/plugin.c
+++ b/src/vlib/unix/plugin.c
@@ -306,6 +306,10 @@ process_reg:
     }
   vec_free (version_required);

+#ifndef RTLD_DEEPBIND
+ #warning "RTLD DEEPBIND is missing from this system and will be ignored!"
+ #define RTLD_DEEPBIND 0
+#endif
   handle = dlopen ((char *) pi->filename,
                   RTLD_LAZY | (reg->deep_bind ? RTLD_DEEPBIND : 0));

